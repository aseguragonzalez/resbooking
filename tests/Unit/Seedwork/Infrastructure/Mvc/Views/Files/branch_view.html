<div>
    <p>
        {{model->name}}
    </p>
    <p>
        {{model->description}}
    </p>
    {{#if model->isBranch():}}
    <p>
        Use a method expression
    </p>
    {{#endif:}}
    {{#if !model->invertBranch():}}
    <p>
        Invert method expression
    </p>
    {{#endif:}}
    {{#if model->isBooleanProperty:}}
    <p>
        Use a boolean property
    </p>
    {{#endif:}}
    {{#if isProperty:}}
    <p>
        Use direct property
    </p>
    {{#endif:}}
    {{#if hasItems:}}
    <div>
        <p>Show if items</p>
        <ul>
            {{#for item in items:}}
            <li>{{item}}</li>
            {{#endfor items:}}
        </ul>
    </div>
    {{#endif:}}
</div>
