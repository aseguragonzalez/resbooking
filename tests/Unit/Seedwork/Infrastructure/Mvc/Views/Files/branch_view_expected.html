<div>
    <p>
        Peter Parker
    </p>
    <p>
        Friendly neighborhood Spider
    </p>
    <p>
        Use a method expression
    </p>
    <p>
        Invert method expression
    </p>
    <p>
        Use a boolean property
    </p>
    <p>
        Use direct property
    </p>
    <div>
        <p>Show if items</p>
        <ul>
            <li>option 1</li>
            <li>option 2</li>
            <li>option 3</li>
        </ul>
    </div>
    <p>
        Use not null property
    </p>
    <p>
        Use of array property (key-value): My Value
    </p>
</div>
